pkg/aws_config_server/list_roles.go

```diff
@@ -160,7 +160,7 @@ func clientRoleMapFromProfile(
 				clientRoleMapping[clientID] = []ConfigProfile{currentConfig}
 				continue
 			}
-			logrus.Debug("function: aws_config_server/list_roles.go/clientRoleMapFromProfile()\n About to append currentConfig to clientRoleMapping")
+			logrus.Debug("function: aws_config_server/list_roles.go/clientRoleMapFromProfile(). About to append currentConfig to clientRoleMapping")
```

Comment:
Headsup that logrus can include method info in its output format: https://github.com/sirupsen/logrus#logging-method-name